dojo/unittests/tools/test_netsparker_parser.py

```diff
@@ -13,36 +13,36 @@ def test_parse_file_with_one_finding(self):
         with self.subTest(i=0):
             finding = findings[0]
             self.assertEqual("Medium", finding.severity)
-            self.assertEqual(16, finding.cwe)
+            self.assertEqual('16', finding.cwe)
```

Comment:
today, `cwe` attribute is a number.

Reply:
@damiencarol I checked it from [this action run](https://github.com/DefectDojo/django-DefectDojo/runs/2912823597). It was failed as this value was a string.

```
Traceback (most recent call last):
2021-06-25T09:49:04.7111523Z [36muwsgi_1         |[0m   File "/app/dojo/unittests/tools/test_netsparker_parser.py", line 16, in test_parse_file_with_one_finding
2021-06-25T09:49:04.7112747Z [36muwsgi_1         |[0m     self.assertEqual(16, finding.cwe)
2021-06-25T09:49:04.7113639Z [36muwsgi_1         |[0m AssertionError: 16 != '16'
```